web/app/themes/dp-timber-theme/components/molecule/Hero/hero.scss

```diff
@@ -0,0 +1,105 @@
+@use "../../SCSS/_variables/index.scss" as var;
+@use "../../SCSS/components/default";
+
+.hero {
+	@include var.displayFlex;
+	@include var.flexCenter;
+	padding: 6em 0 2em 0;
+	background-attachment: fixed;
+	@include var.background;
+	border-bottom: 0 !important;
+	height: 60vh;
+	margin-bottom: -3.25em;
+	max-height: 32em;
+	min-height: 22em;
+	position: relative;
+	top: -3.25em;
+	color: #fff;
+
+	.inner {
+		padding: 4em 0 2em 0;
+		margin: 0 auto;
+		max-width: 65em;
+		width: calc(100% - 6em);
+		z-index: 2;
+
+		.content {
+			display: flex;
+			align-items: center;
+			margin: 0 0 2em 0;
+
+			p {
+				font-size: 0.7em;
+				font-weight: 600;
+				letter-spacing: 0.25em;
+				text-transform: uppercase;
+				margin-right: 1.5em;
+			}
+		}
+	}
+
+	&.major {
+		height: 75vh;
+		min-height: 30em;
+		max-height: 50em;
+	}
+
+	&::after {
+		content: "";
+		@include var.transition-opacity(0.25s);
+		pointer-events: none;
+		background-color: var.$ebony-clay;
+		display: block;
+		height: 100%;
+		left: 0;
+		opacity: 0.85;
+		position: absolute;
+		top: 0;
+		width: 100%;
+		z-index: 1;
+	}
+}
+
+@media screen and (max-width: 736px) {
+	.hero {
+		padding: 5em 0 1em 0;
+		height: auto;
+		margin-bottom: -2.75em;
+		max-height: none;
+		min-height: 0;
+		top: -2.75em;
+	}
+
+	.hero h1 {
+		font-size: 2em;
+	}
+
+	.hero>.inner .content {
+		display: block;
+	}
+
+	.hero>.inner .content>* {
+		margin-right: 0;
+		margin-bottom: 2em;
+	}
+
+	.hero.major {
+		height: auto;
+		min-height: 0;
+		max-height: none;
+	}
+}
+
+@media screen and (max-width: 480px) {
+	.hero {
+		padding: 6em 0 2em 0;
+	}
+
+	.hero>.inner .content p br {
```

Comment:
Need to break this.

Reply:
I found this code not usefull, therefore I removed it